### PR TITLE
NRE Mission Math Fixes #62521

### LINKED ISSUE #62521: Robofac NRE maths are really weird and heavily favor short printouts
### Is your feature request related to a problem? Please describe.

NRE recorder requires a specific number of beeps to generate a printout. 1-2 beeps generate a short printout, 3-24 generate a medium printout, and 25+ generate a long printout. Robofac pays 4 HGC for a short printout, 5 HGC for a medium printout and 12 for a large printout. They will buy these printouts repeatedly.

Therefore, making short printouts generates 4 HGC per beep, making medium printouts generates 1.67 HGC per beep, and making long printouts generates .46 HGC per beep.

Short printouts are the least useful data by far so it is weird that they are the most efficient way to generate money by giving Robofac NRE printouts.

### Solution you would like.

Either the payout should change such that making the long printout (the riskiest and hardest data to generate) is the most profitable and/or there should be a limit on how many short/medium printouts are made as Robofac may only need so many of the short printouts before getting more is just redundant.

### Describe alternatives you have considered.

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
#62521

#### Describe the solution
Adjusts the Hubcoin reward to make more sense.  

#### Describe alternatives you've considered
Using a python script to multiply all PA prices by 5. Then evaluating the rewards of this mission.

#### Testing
NA

### CODE REVIEW

**Comment by Ilysen** [comment]
Make sure you update the mission text and dialogue too - I think they specifically mention the reward amounts

**Comment by Maleclypse** [comment]
> Make sure you update the mission text and dialogue too - I think they specifically mention the reward amounts

Could you double check that because when I looked earlier and now again, it mentions "agreed upon price" but I can't find anywhere they told you ahead of time what you'd get.

**Comment by Maleclypse** [comment]
> Make sure you update the mission text and dialogue too - I think they specifically mention the reward amounts

OK I think the only price mentioned is for the small ones and that larger ones will receive bonus payments.

File: data/json/npcs/robofac/robofac_intercom/robofac_intercom_missions.json [hunk 1]
```diff
@@ -829,7 +829,7 @@
       "MISSION_ROBOFAC_INTERCOM_PORTAL_SM_1_OFFER_ADVICE_Replacement"
     ],
     "type": "talk_topic",
-    "dynamic_line": "As we mentioned before, our research into the portal storm phenomena requires that we obtain additional data to verify our own local measurements.  Recording this data will require you to carry a scientific instrument called an NRE recorder on your person while outside in the storm for around six minutes.  The recording instrument will be provided at no cost if you choose to accept.\"\n\n\"We offer the standard hazard pay of 4 HGC for completion of this contract.",
+    "dynamic_line": "As we mentioned before, our research into the portal storm phenomena requires that we obtain additional data to verify our own local measurements.  Recording this data will require you to carry a scientific instrument called an NRE recorder on your person while outside in the storm for around six minutes.  The recording instrument will be provided at no cost if you choose to accept.\"\n\n\"We offer the standard hazard pay of 1 HGC for completion of this contract.",
```
Comment (Ilysen) [inline data/json/npcs/robofac/robofac_intercom/robofac_intercom_missions.json:832]:
```suggestion
    "dynamic_line": "As we mentioned before, our research into the portal storm phenomena requires that we obtain additional data to verify our own local measurements.  Recording this data will require you to carry a scientific instrument called an NRE recorder on your person while outside in the storm for around six minutes.  The recording instrument will be provided at no cost if you choose to accept.\"\n\n\"We are willing to offer 1 HGC for each standard-length recording.  Higher-quality data may be obtained by recording for longer lengths of time, and as a result, we are also willing to provide higher pay for such recordings.",
```

**Comment by Ilysen** [comment]
There are also some other dialogue lines that could use some updating, specifically the trade-in dialogue for the medium and long recording types, in `robofac_intercom_trades.json`, lines 113 and 132. 113 mentions "the agreed payment, and a small bonus", and 132 specifically mentions triple the agreed payment, which would be made inaccurate by this PR.

**Comment by Ilysen** [comment]
I'm not sure I can leave comments on later chunks of the file that didn't get changed. Heck.

**Pull request merged**